data/transactions/logic/opcodes.go

```diff
@@ -657,6 +657,7 @@ var OpSpecs = []OpSpec{
 	{0x84, "ed25519verify_bare", opEd25519VerifyBare, proto("bb{64}b{32}:T"), 7, costly(1900)},
 	{0x85, "falcon_verify", opFalconVerify, proto("bb{1232}b{1793}:T"), 12, costly(1700)}, // dynamic for internal hash?
 	{0x86, "sumhash512", opSumhash512, proto("b:b{64}"), sumhashVersion, costByLength(150, 7, 4, 0)},
+	{0x87, "sha512", opSHA512, proto("b:b{64}"), 13, costByLength(15, 32, 2, 0)},
```

Comment:
Is it worth renaming `sumhashVersion and applying here rather than explicit 13?

Reply:
I'm more inclined to do that when I'm unsure when it'll go in. But I think we're confident this goes in at the next protocol bump.

My reasoning that I'm slightly against these things existing as named values. It gives the impression they can be changed. But once they are really in protocol, changing the constant would break things.  But for things that we aren't sure will go in, the named value is _good_ because we _can_ still change them.